poleno/attachments/models.py

```diff
@@ -57,8 +57,8 @@ class Attachment(FormatMixin, models.Model):
     # May NOT be empty: Automatically sanitized in save() when creating a new object.
     name = models.CharField(max_length=255,
             help_text=squeeze(u"""
-                Attachment file name, e.g. "document.pdf". Automatically sanitized when
-                creating a new object.
+                Attachment file name, e.g. "document.pdf". Automatically sanitized when creating
```

Comment:
*Tento komentar sa tyka `misc/attachments.odt`, ale ku binarnemu suboru sa nedaju pridavat komentare*

@viliambalaz Field `content_type` je klasicka property, kedze je v DB. Computed properties su take, ktore nie su v DB ale su dopocitane "on the fly" pri po selectnuti instancie.

Trochu som preformuloval texty pri `name` a `content_type` fieldoch. Dokumentacia nepise ako je vec implementovana (hodnota sa vypocita v `save()`) alebo co to robi (hodnota sa vypocita pri vytvarani instancie).